.gitignore

```diff
@@ -1 +1,4 @@
+.idea
+.DS_Store
+
```

Comment:
could you remove these unrelated changes? 

I recommend to create `.gitignore` folder in your user’s home directory, usually `~/.gitignore` where you can add these lines